pkg/scaffold/v2/controller.go

```diff
@@ -82,6 +82,7 @@ import (
 type {{ .Resource.Kind }}Reconciler struct {
 	client.Client
 	Log logr.Logger
+    Scheme *runtime.Scheme
```

Comment:
spacing is off here.  Maybe a tabs vs spaces issue?

Reply:
Fix. thank you.
This diff of layout did not appears in the idea. Have any specific/setup config that is used by you folks?